task2-bash/backup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+
+src_folder="../task1-simple-program"
+tmp_folder="tmp/guesanumber"
+
+
+## Copying files to tmp folder
+mkdir -p $tmp_folder
+cp $src_folder/*.{c,h} $tmp_folder
+
+## Archiving backup to release folder
+mkdir -p $src_folder/release
+tar --directory=$(dirname $tmp_folder) -cvzf \
+	$src_folder/release/$(basename $tmp_folder).tar.gz \
+	$(basename $tmp_folder)
+	
```

Comment:
Please don't forget to add this newline in all files

Reply:
Yes. This has already been fixed in subsequent commits